lib/segment/src/index/struct_payload_index.rs

```diff
@@ -68,7 +68,7 @@ impl StructPayloadIndex {
             };
             let mut result_estimation: Option<CardinalityEstimation> = None;
             for index in indexes {
-                result_estimation = index.estimate_cardinality(&full_path_condition);
+                result_estimation = index.estimate_cardinality(&full_path_condition).ok();
```

Comment:
🤔 decided to go with the most nave approach here

Reply:
I think it is fine to use `.ok()` here.